docs/cloud-messaging/receive.md

```diff
@@ -409,3 +409,97 @@ experimentalSetDeliveryMetricsExportedToBigQueryEnabled(messaging, true);
 ```
 
 Don't forget to run `yarn build` in order to export the new version of your service worker to the `web` folder.
+
+
+## (Advanced, Optional) Allowing Notification Images on iOS
+
+On Apple devices, in order for incoming FCM Notifications to display images from the FCM payload, you must add an additional notification service extension. This is not a required step.
+
+If you are using FirebaseAuth phone authentication, you must add the Firebase Auth pod to your Podfile.
+
+### Step 1 - Add a notification service extension
+
+- From Xcode top menu go to: File > New > Target...
```

Comment:
We generally use boldface for UI elements.  Suggest bolding "File > New > Target" and all other UI elements in this PR.